<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>galgamedialog</class>
 <widget class="QWidget" name="galgamedialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>200</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Microsoft YaHei</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
    background: transparent;
}
</string>
  </property>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>15</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>39</y>
     <width>600</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::NoFrame</enum>
   </property>
   <property name="acceptRichText">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>说点什么吧（Shift+Enter回车 Enter发送）</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>601</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_input">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>长按输入</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_autoInput">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>80</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>直接对话</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_history">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>170</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>历史</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
